### PR TITLE
parser: Support drop tables

### PR DESCRIPTION
**Pull request by zimulala**
#176

### CODE REVIEW

**Comment by qiuyesuifeng** [comment]
LGTM

File: parser/parser.y [hunk 1]
```diff
@@ -1322,6 +1322,10 @@ DropTableStmt:
 		}
 	}
 
+DropTableOpt:
```
Comment (shenli) [inline parser/parser.y]: TableOrTables is a better name.
This is not an option or ignorable token.
MySQL yacc file use table_or_tables.

Comment (zimulala) [inline parser/parser.y]: gotcha

**Comment by zimulala** [comment]
PTAL @shenli

**Comment by shenli** [comment]
LGTM

**Pull request merged**